js/script.js

```diff
@@ -0,0 +1,68 @@
+
+const userInput = document.querySelector("#exampleDataList");
+const searchBtn = document.querySelector("#button-addon2");
+const searchBox = document.querySelector(".searchInputBox")
+const list = document.querySelector(".resultList");
+const loadingSpinner = document.querySelector(".loadingSpinner");
+const marqueeDiv = document.querySelector(".marqueeTextContainer");
+const marqueeLoading = document.querySelector(".loadingMarquee");
+let image;
+
+searchBtn.addEventListener('click', generateSearchList)
+
+function generateSearchList() {
+    list.innerHTML = "";
+    const query = userInput.value;
+    if (!query) {
+        return;
+    }
+    enableSpinner(loadingSpinner)
+    fetch(`https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${query}&limit=10&exchange=NASDAQ`)
+        .then(response => response.json())
+        .then(data => {
+
+            for (let i = 0; i < data.length; i++) {
```

Comment:
nice. consider using promise.all() for getting the data faster and manipultaing the DOM only once